bin/docker-image-tool.sh

```diff
@@ -227,6 +227,7 @@ Examples:
 EOF
 }
 
+# shellcheck disable=SC2199
```

Comment:
this is an acceptable approach, we can disable the check first, and fix them one by one in the future. (if the fix is simple, it's fine to fix in place in this PR as well)